app/services/normalise_charge.js

```diff
@@ -1,15 +1,15 @@
 'use strict'
 
-// NPM dependencies
+// npm dependencies
```

Comment:
```suggestion
// NPM dependencies
```
Not sure why this changed to lower case?

Reply:
I uncapitalised the comments to be consistent with:
- https://github.com/alphagov/pay-frontend/blob/43a00b9eba7fc8fee389c83b5bef6ef60b11c3eb/app/middleware/resolve_service.js#L3
- https://github.com/alphagov/pay-frontend/blob/master/app/utils/charge_validation_fields.js#L6
- https://github.com/alphagov/pay-frontend/blob/master/app/utils/charge_validation_backend.js#L3
- https://github.com/alphagov/pay-frontend/blob/master/app/utils/charge_validation.js#L3
etc

I was not aware about https://eslint.org/docs/rules/capitalized-comments , thanks for pointing this out 👍